src/components/Events.js

```diff
@@ -1,37 +1,50 @@
 import React from 'react'
-import { Link } from 'gatsby'
+import { StaticQuery, graphql } from 'gatsby'
+import EventCard from './EventCard'
 
-export const Events = () => (
-    <section className="event-list clearfix">
-        <div className="wrapper">
-            <article className="active">
-                <span className="month">August</span>
-                <a href="https://www.meetup.com/en-AU/Front-End-Web-Developers-Perth/events/qqvhjkyxlblb/" title="August Casual Catchup">
-                    <span className="tag social icon">Social</span>
-                    <strong>Casual Catchup</strong>
-                    <span className="date">Wednesday, August 8</span>
-                </a>
-            </article>
+const Events = () => (
+    <StaticQuery
+        query={graphql`
+            query EventsQuery {
+                eventList: dataJson {
+                    events {
+                        active
+                        type
+                        title
+                        date(formatString: "dddd, MMMM D")
+                        month: date(formatString: "MMMM")
+                        url
+                    }
+                }
+            }
+        `}
+        render={data => {
+            const usedMonths = []
```

Comment:
I can't see where else this is used? The following code will push a momnth to the array if it is not previously added, but then the variable is not referenced anywhere else?

Reply:
Used here. The design only has the month the first time. So checks if it's there, then adds to here and continues on.
https://github.com/fendersperth/fenders-website/blob/e6614ff0c39ec896dbae87e54985cbe16be3bb95/src/components/Events.js#L29

I had hoped Graphql would be able to leave it out after the first instance but I couldn't see a way of doing that.